requirements.txt

```diff
@@ -1 +1,2 @@
+pyyaml
```

Comment:
What do you think about pinning the version here?

Reply:
done